File: SmartMenza.Business/Models/Statistics/DishStatisticsResponse.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SmartMenza.Business.Models.Statistics
{
    public class DishStatisticsResponse
    {
        public int DishId { get; set; }
        public string Title { get; set; } = string.Empty;
        public string? ImgPath { get; set; }
        public double? AverageRating { get; set; }
        public int RatingsCount { get; set; }
        public int FavoriteCount { get; set; }
    }
}

File: core/src/main/java/foi/cverglici/core/data/model/employee/statistics/DishStatisticsResponse.kt
package foi.cverglici.core.data.model.employee.statistics

import com.google.gson.annotations.SerializedName

data class DishStatisticsResponse(
    @SerializedName("dishId") val dishId: Int,
    @SerializedName("title") val title: String,
    @SerializedName("imgPath") val imgPath: String?,
    @SerializedName("averageRating") val averageRating: Double?,
    @SerializedName("ratingsCount") val ratingsCount: Int,
    @SerializedName("favoriteCount") val favoriteCount: Int
)